src/app/api/quality_matrix/quality_matrix.py

```diff
@@ -49,8 +49,161 @@ def all_sources() -> dict[str, int]:
     )
 
 
+def default_properties() -> list[str]:
+    return [
+        "virtual:collection_id_primary",
+        "virtual:collection_id",
+        "ccm:collection_proposal_status",
+        "virtual:portal_type",
+        "ccm:oeh_quality_correctness",
+        "ccm:published_handle_id",
+        "sys:node-uuid",
+        "cm:versionLabel",
+        "virtual:search_tasks",
+        "virtual:author",
+        "virtual:time_after_modified",
+        "virtual:time_after_created",
+        "virtual:editorial_exclusion",
+        "virtual:editorial_taxonid",
+        "virtual:editorial_file_type",
+        "virtual:editorial_edit_type",
+        "virtual:editorial_filter",
+        "virtual:editorial_status",
+        "virtual:publish_location",
+        "virtual:data_privacy",
+        "virtual:newsletter",
+        "virtual:email",
+        "ccm:lifecyclecontributer_publisher",
+        "ccm:replicationsource",
+        "ccm:general_identifier",
+        "ccm:wwwurl",
+        "ccm:oeh_digitalCompetencies",
+        "ccm:oeh_languageLevel",
+        "cclom:aggregationlevel",
+        "ccm:curriculum",
+        "ccm:lifecyclecontributer_author",
+        "ccm:author_freetext",
+        "ccm:custom_license",
```

Comment:
[another](https://tenor.com/view/another-thor-chrishemsworth-gif-4736083)

Reply:
-_- +#"§$"§$ I recreated the list from a set() - no more duplicates :D